usaspending_api/etl/subaward_etl.py

```diff
@@ -180,9 +180,16 @@ def load_subawards(submission_attributes, db_cursor):
         recipient = LegalEntity.objects.filter(recipient_unique_id=row['duns']).first()
 
         if not recipient and row['duns']:
+            recipient_name = row['awardee_name']
+            if recipient_name is None:
+                recipient_name = row['awardee_or_recipient_legal']
+            if recipient_name is None:
+                logger.warn("Subaward number {} has no recipient name; skipping...".format(row['subaward_num']))
```

Comment:
Should we be skipping the row entirely or just loading it with a blank recipient name? `recipient_name` on the `LegalEntity` model is kinda strange...it allows `blank=True` but doesn't allow NULLS. 

You've probably already discussed with @kaitlin  the ramifications of skipping a subaward record, but wanted to double check, since we're crossing a line into territory that would affect dollar amounts.